api/graphql/reservation_units/reservation_unit_types.py

```diff
@@ -262,7 +262,7 @@ def resolve_category(self, info):
         return self.category
 
 
-class ApplicationRoundType(AuthNode, PrimaryKeyObjectType):
+class ReservationUnitApplicationRoundType(AuthNode, PrimaryKeyObjectType):
```

Comment:
I think we can get rid of this and use the new application round type.

Reply:
Removed and functionality ported to new Type. (That functionality that did not exist yet)